cmd/snapshot-controller/main.go

```diff
@@ -73,7 +75,7 @@ var (
 	retryIntervalMax              = flag.Duration("retry-interval-max", 5*time.Minute, "Maximum retry interval of failed volume snapshot creation or deletion. Default is 5 minutes.")
 	enableDistributedSnapshotting = flag.Bool("enable-distributed-snapshotting", false, "Enables each node to handle snapshotting for the local volumes created on that node")
 	preventVolumeModeConversion   = flag.Bool("prevent-volume-mode-conversion", true, "Prevents an unauthorised user from modifying the volume mode when creating a PVC from an existing VolumeSnapshot.")
-	enableVolumeGroupSnapshots    = flag.Bool("enable-volume-group-snapshots", false, "Enables the volume group snapshot feature, allowing the user to create a snapshot of a group of volumes.")
+	//enableVolumeGroupSnapshots    = flag.Bool("enable-volume-group-snapshots", false, "Enables the volume group snapshot feature, allowing the user to create a snapshot of a group of volumes.")
```

Comment:
Not sure what usually is done, but deprecating an option usually involves keeping the it, but ignoring it. That prevents errors when a deployment has it set, and the container-image is updated.

Reply:
it is showing a warning if we keep this variable because it is no more used anywhere. So I think it would be fine removing this

Reply:
Let's remove this line if we don't have the `enable-volume-group-snapshots` flag.

Reply:
I agree with @nixpanic and I'd like to avoid breaking existing deployments.
If we found it specified, we could warn the user that the option is deprecated.